### PR TITLE
docs: add markdown-magic to automatically add project

### LINKED ISSUE #52: [README.md] - Automatic Generation of Projects in README.md/PROJECT_SUPPORTED.md
As of now, in order to add projects we do 2 things:
1. add project in `data/projects.json`
2. update `README.md`

Step-2 might get forgotten sometime. This can be automated stuff as part of `npm script`.

We can have `PROJECT_SUPPORTED.md` where we list all the projects.

@bnb @maddhruv Do let me know your thoughts.

### PR DESCRIPTION
**Pull request by DSchau**
Fixes #52

This (small) PR adds markdown-magic which will automatically add projects to README.md when the `markdown` script is run. I've also updated the instructions on adding a project to use this script.

### CODE REVIEW

**Comment by bnb** [comment]
Will review this soon 👍

**Comment by hemal7735** [comment]
This PR addresses the Issue #52 **Automatic Generation of Projects in README.md/PROJECT_SUPPORTED.md**

Thank you @DSchau ! You are awesome 🌟

File: README.md [hunk 1]
```diff
@@ -55,17 +55,23 @@ If you'd like to help with any of these, feel free to submit a PR or ask how you
 
 The table of projects which are _currently_ supported.
 
+<!-- AUTO-GENERATED-CONTENT:START (PROJECTS:path=./data/projects.json) -->
```
Comment (hemal7735) [inline README.md]: @DSchau what are your thoughts on adding some sort of `WARNING` comment here?

Comment (DSchau) [reply README.md]: Updated!

File: README.md [hunk 2]
```diff
@@ -74,8 +84,8 @@ If you'd like to add a new project to `good-first-issue`, you're more than welco
 - Update `data/projects.json`
   - Add your `<project>` as a property of `projects` with an object that includes a `name`, `description`, and a `q` (representing the GitHub search query).
 
-- Update `README.md`
-  - Add your `<project>` at the bottom of the [Projects](#projects) table above with `name` and `project`
+- Update `README.md` by running `npm run markdown`
```
Comment (bnb) [inline README.md]: This is awesome. I'm curious: do you think we should add a CI check to see if this is done?

**Pull request merged**